boa_engine/src/builtins/date/mod.rs

```diff
@@ -137,23 +120,26 @@ impl BuiltIn for Date {
         .method(Self::set_utc_minutes, "setUTCMinutes", 3)
         .method(Self::set_utc_month, "setUTCMonth", 2)
         .method(Self::set_utc_seconds, "setUTCSeconds", 2)
+        .method(Self::set_year, "setYear", 1)
         .method(Self::to_date_string, "toDateString", 0)
-        .method(getter_method!(to_gmt_string), "toGMTString", 0)
+        .method(Self::to_gmt_string, "toGMTString", 0)
         .method(Self::to_iso_string, "toISOString", 0)
+        // Why toJSON length is 1, does `method()`'s length signifies number of
+        // inputs? If so shouldn't it be 0?
```

Comment:
Alright, I should also check if other functions are listed correctly.

Reply:
By the way I see that you have changed the legth for the toLocale* functions to 2. Not really releveant, because we want to remove them, but just for your info; length 0 would be correct for those. The spec specifies their siganures like this: [Date.prototype.toLocaleDateString ( [ reserved1 [ , reserved2 ] ] )](https://tc39.es/ecma262/#sec-date.prototype.tolocaledatestring). Notice that the wrapping the arguments in square brackets means that they are optional. Optional arguments do not count towards the length of the function.